README.md

```diff
@@ -18,6 +18,13 @@ This list is for developers who are looking for non-trivial quality applications
 ## Web Apps (frontend only)
 ----
 
+### [Codesandbox](https://github.com/CompuIves/codesandbox-client)
+
+* react, ngix, phoenix
```

Comment:
Do you mean `nginx`? Or is ngix some js package or tool I'm not aware of?

Reply:
nginx. Sorry I will fix this.

Reply:
@mickael-kerjean 010001